packages/react/src/primitives/SliderField/SliderField.tsx

```diff
@@ -6,12 +6,13 @@ import { ComponentClassNames } from '../shared/constants';
 import { FieldDescription, FieldErrorMessage } from '../Field';
 import { FieldGroup } from '../FieldGroup';
 import { Flex } from '../Flex';
-import { isFunction, useStableId } from '../shared/utils';
```

Comment:
Non-blocking: should we just have one `utils` instead of `shared/utils` and `utils/`?

Reply:
My goal is to move all the utils functions out of shared and to a `utils/` folder. I can do that in a follow up PR